### PR TITLE
feature: add the upload question page for question hub section

### PR DESCRIPTION
**Pull request by YuetongLiiii**
Hi, I currently finish the upload question page, Please have a review the layout. If there is no problem I will merge it

### CODE REVIEW

File: itp-front/src/assets/image/UploadQuestion.png
Comment (Leshangpang) [inline itp-front/src/assets/image/UploadQuestion.png:1]: Note that the aim of the website is to learn about all the 17 SDGs. We are only implementing the functionality for goal 5 only but the website is still about the SDGs and not just gender equality. Thus, for this page, it is once again for all 17 goals not just goal 5, so this image may not be so relevant.

**Review by Leshangpang — CHANGES_REQUESTED** [review]
I think the input boxes can be larger to better breathability. Consider adding some space between boxes and spacing between the heading and the input box. And in terms of texts inside boxes such as "Please Fill the Choice", maybe refer to how it is done on other platforms such as google form. 
Maybe reconsider the colour of the buttons, please try to avoid using new colours.

**Review by Wj12344 — CHANGES_REQUESTED** [review]
I reviewed your upload question page, and overall, it looks great. However, the text "Write your Question" at the top isn't very visible. Could you change it to a more noticeable color? Also, the font size seems a bit small, so it would be better to increase it slightly.

**Review by Leshangpang — APPROVED** [review]
I think this background looks much better than the previous one. Maybe consider changing the instructions in the box like "Option 1" instead of "Please fill xxx". Regardless, the current version is looking good and I think it is ready for merge.

**Review by Wj12344 — APPROVED** [review]
Good work! Better design than before, and the background image is nice! ^_^

**Pull request merged**